lineman/app/components/thread_page/activity_card/new_motion.haml

```diff
@@ -1,9 +1,9 @@
-.thread-item{ng-controller: 'NewProposalItemController'}
+%article.thread-item{ng-controller: 'NewProposalItemController', aria-labelledby: '{{proposal.id}}'}
```

Comment:
yea.. so a proposal.id could easily clash with a comment.id .. so you need to namespace the value with the type. aria-labelledby: 'proposal-{{proposal.id}}'